yang/frr-zebra.yang

```diff
@@ -253,6 +279,37 @@ module frr-zebra {
 
     uses route-common;
   }
+  // End of ip6-route
+
+  /*
+   * IP Prefix Route object.
+   */
+
+  grouping ip-route {
+    description
+      "An IPv4/IPv6 prefix route.";
+    leaf prefix {
+      type inet:ip-prefix;
+      description
+        "The route's prefix.";
+    }
+
+    leaf protocol {
+      when "'../../afi-safi-name' = 'ipv4-unicast'";
```

Comment:
Don't think this should be in a grouping if you're referencing leaves outside the grouping. If you know a leaf is going to be there just kill the grouping and put it all in the spot where the grouping is used.